athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/DocDBMetadataHandler.java

```diff
@@ -208,15 +209,16 @@ public ListTablesResponse doListTables(BlockAllocator blockAllocator, ListTables
      * @param request
      * @return
      */
-    private List<TableName> doListTablesWithCommand(MongoClient client, ListTablesRequest request)
+    private List<String> doListTablesWithCommand(MongoClient client, ListTablesRequest request)
     {
         logger.debug("doListTablesWithCommand Start");
-        List<TableName> tables = new ArrayList<>();
-        Document document = client.getDatabase(request.getSchemaName()).runCommand(new Document("listCollections", 1).append("nameOnly", true).append("authorizedCollections", true));
+        List<String> tables = new ArrayList<>();
+        Document queryDocument = new Document("listCollections", 1).append("nameOnly", true).append("authorizedCollections", true);
+        Document document = client.getDatabase(request.getSchemaName()).runCommand(queryDocument);
 
         List<Document> list = ((Document) document.get("cursor")).getList("firstBatch", Document.class);
         for (Document doc : list) {
-            tables.add(new TableName(request.getSchemaName(), doc.getString("name")));
+            tables.add(doc.getString("name"));
         }
```

Comment:
Since you turn this into a stream in the only place that uses this anyway, we can simplify this to:

```
    private Stream<String> doListTablesWithCommand(MongoClient client, ListTablesRequest request)
    {
        logger.debug("doListTablesWithCommand Start");
        Document document = client.getDatabase(request.getSchemaName()).runCommand(new Document("listCollections", 1).append("nameOnly", true).append("authorizedCollections", true));
        List<Document> list = ((Document) document.get("cursor")).getList("firstBatch", Document.class);
        return list.stream().map(doc -> doc.getString("name")).sorted();
    }
```
```